src/v2/view-builder/views/webauthn/EnrollWebauthnView.js

```diff
@@ -20,7 +20,9 @@ function getExcludeCredentials(authenticatorEnrollments = []) {
 }
 
 const Body = BaseForm.extend({
-  title: loc('oie.enroll.webauthn.title', 'login'),
+  title() {
+    return loc('oie.enroll.webauthn.title', 'login');
```

Comment:
Why does this need to be a function now?

Reply:
This will fix the i18n leak issue. I think there is some timing issue if we are using it as properties

Reply:
Looks like we also do this elsewhere, as the English value is loaded in the browser by default - and returning a `function` here ensures the `loc` method gets executed a bit later.

There are quite a few occurrences where we're mapping this attribute and others (subtitle, save, etc) to a `string`, so @nikhilvenkatraman-okta what do you think about creating a lint rule for this?